ibis/backends/base/sql/alchemy/registry.py

```diff
@@ -422,7 +425,7 @@ def _sort_key(t, expr):
     ops.Min: reduction(sa.func.min),
     ops.Max: reduction(sa.func.max),
     ops.CountDistinct: _count_distinct,
-    ops.GroupConcat: reduction(sa.func.group_concat),
+    ops.GroupConcat: _group_concat,
```

Comment:
I guess this can't be a reduction because of the `sep` argument?

Reply:
Correct.